app/main.py

```diff
@@ -1,7 +1,8 @@
 def copy_file(command: str) -> None:
     command_ls = command.split()
     try:
-        with open(command_ls[1], "r") as file_in, open(command_ls[2], "x") as file_out:
+        with (open(command_ls[1], "r") as file_in,
+              open(command_ls[2], "x") as file_out):
             file_out.write(file_in.read())
     except FileExistsError:
         pass
```

Comment:
Try to avoid passing in except blocks.

Reply:
I'm not quite understand what you want me to do, cause we need to do nothing if we try to create file with the same name, so we need try except block. If you know another solution, give me a tip, so i can figure it out on my own.

Reply:
'passing in except' can lead to silent errors and make it difficult to identify and handle exceptions properly. Error handling means processing them in a certain way.